packages/server/src/modules/auth/auth.resolver.ts

```diff
@@ -112,7 +106,8 @@ export class AuthResolver {
     const user = await this.userService.findUserByEmail(email);
     // To prevent user enumeration attacks, we don't throw an error
     // TODO: it can still be guessed with the response time difference, add a relevant delay
-    if (!user) {
+    // TODO: if the deletion is in the grace period, should we send an email explaining the situation or just ignore it?
```

Comment:
For now let's ignore it, in the future we can come back at it and support this use case